### ISSUE TITLE
md5 is not match

### ISSUE THREAD

**Issue by mohamedalykarim** [Gictorbit/photoshopCClinux#104]
In downloading photoshopCC-V19.1.6-2018x64.tgz step, and after the file downloaded successfully I got "md5 is not match" message

**Comment by nickyvdz-art** [comment]
yep, same deal here for me today; maybe connection dropout downloading? ill try later tonight when my net is more stable

**Comment by manoscham** [comment]
same issue here "md5 is not match"
error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz

**Comment by LightDani** [comment]
try use cloudflare warp (1.1.1.1). that's solved mine

**Comment by DeepakGupta001** [comment]
how to fix md5 not match

**Comment by DennisGicheru** [comment]
MD5 not match, how i resolve this issue?

**Comment by amit-nadcab** [comment]
> same issue here "md5 is not match" error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz

got same issue.

**Comment by 0-Vanes-0** [comment]
The error message looks like:
`curl: (92) HTTP/2 stream 0 was not closed cleanly: INTERNAL_ERROR (err 2)`
`md5 is not match`

It tries to download 3 times and then throws:
`error: sorry something went wrong during download photoshopCC-V19.1.6-2018x64.tgz`

> try use cloudflare warp (1.1.1.1). that's solved mine

I've managed to install the vpn, turn it on, test it (all ok).
The guide I used: https://www.poorna.dev/post/cloudflare-warp-for-linux

**BUT STILL the installation fails.**
